### PR TITLE
[CODEOWNERS] Assign CDK owners

### PR DESCRIPTION
**Pull request by jsquire**
# Summary

The focus of these changes is to add the owners for the CDK.  Ordering of existing entries as also updated to fix labels that were not following the alphabetical ordering.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -237,6 +243,11 @@
 # ServiceLabel: %Cognitive - Health Insights
 # ServiceOwners:                                                   @asaflevi-ms
 
+# PRLabel: %Cognitive - Personalizer
+/sdk/personalizer/                                                 @orenmichaely @tyclintw
+
+# ServiceLabel: %Cognitive - Personalizer
+# ServiceOwners:                                                   @orenmichaely @tyclintw
```
Comment (jsquire) [inline .github/CODEOWNERS:250]:
```suggestion
# ServiceOwners:                                                   @orenmichaely @tyclintw

```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -188,7 +194,7 @@
 /sdk/contentsafety/                                                @bowgong @mengaims @JieZhou000
 
 # ServiceLabel: %Cognitive - Content Safety
-# ServiceOwners:                     	                           @bowgong @mengaims @JieZhou000
+# ServiceOwners:                     	                             @bowgong @mengaims @JieZhou000
```
Comment (christothes) [inline .github/CODEOWNERS:197]: nit: why the extra indent here?

Comment (jsquire) [reply .github/CODEOWNERS:197]: Unintentional!

**Review by christothes — APPROVED** [review]

File: .github/CODEOWNERS (hunk 2 above)
Comment (jsquire) [inline .github/CODEOWNERS:197]:
```suggestion
# ServiceOwners:                     	                            @bowgong @mengaims @JieZhou000
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -188,7 +194,7 @@
 /sdk/contentsafety/                                                @bowgong @mengaims @JieZhou000
 
 # ServiceLabel: %Cognitive - Content Safety
-# ServiceOwners:                     	                           @bowgong @mengaims @JieZhou000
+# ServiceOwners:                     	                            @bowgong @mengaims @JieZhou000
```
Comment (jsquire) [inline .github/CODEOWNERS:197]:
```suggestion
# ServiceOwners:                     	                           @bowgong @mengaims @JieZhou000
```

**Pull request merged**